connect/proxy/config.go

```diff
@@ -242,6 +248,9 @@ func (w *AgentConfigWatcher) handler(blockVal watch.BlockingParamVal,
 	}
 	cfg.PublicListener.BindAddress = resp.Address
 	cfg.PublicListener.BindPort = resp.Port
+	if resp.Proxy.LocalServiceSocketPath != "" {
+		panic(fmt.Sprintf("Unhandled unix domain socket config %+v %+v", resp.Proxy, cfg.PublicListener))
```

Comment:
There's precedent earlier in this function to handle errors this way:

```suggestion
		w.logger.Error("Unhandled unix domain socket config", "proxy", resp.Proxy, "listener", cfg.PublicListener)
		return
```